app/modules/intelligence/agents_copy/chat_agents/supervisor_agent.py

```
@@ -0,0 +1,67 @@
+                        prompt_provider,
+                        rag_agent=blast_radius_agent.BlastRadiusAgent(
+                            llm_provider, tools_provider, project_id
+                        ),
+                        agent_type=AgentType.CODE_CHANGES,
+                    ),
+                ),
+                AgentWithInfo(
+                    id="code_review_agent",
+                    name="Code Review Agent",
+                    description="An agent specialized in code reviews, use this for reviewing codes",
+                    agent=simple_llm_agent,
```

Ruff (0.8.2)

Undefined name `simple_llm_agent`

(F821)